src/Employer/Employer.Web/Views/Shared/_Layout.cshtml

```diff
@@ -90,7 +90,7 @@ public string GetTitle()
 
     <div asp-show="@ViewContext.ViewData[ViewDataKeys.CanShowOutageMessage] as bool?" id="global-outage-message" class="global-message">
         <p>@ViewContext.ViewData[ViewDataKeys.PlannedOutageMessage]</p>
-        <p><a asp-route="@RouteNames.DismissOutageMessage_Get">Dismiss</a></p>
+        <p><a asp-route="@RouteNames.DismissOutageMessage_Get" asp-route-returnUrl="@ViewContext.HttpContext.Request.Path">Dismiss</a></p>
```

Comment:
Could we encrypt the returnUrl? Then if it decrypts OK then we know its one of ours and don't need to do the regex.

Reply:
If we turn this to a POST then we'd have the CSRF security of the Anti-forgery token.

Reply:
Will change to use POST 👍